### PR TITLE
feat: added snackbar message component

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
snackbar-preview

### CODE REVIEW

File: components/sanckbar.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import url(/stylesheets/main.css);
+
+.snackbar-container{
+    width: 100%;
+    background-color: hsl(var(--primary-lighter));
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (nakulsharma15) [inline components/sanckbar.css:8]: You can extract this in a utility class so that it can be used elsewhere as well

**Review by nakulsharma15 — COMMENTED** [review]
Suggested a minor change

File: components/sanckbar.css (hunk 1 above)
Comment (vaibhav-mahalle) [reply components/sanckbar.css:8]: will work on that while refactoring the code.

Comment (nishasen) [inline components/sanckbar.css:8]: use utility class here instead

**Review by nishasen — APPROVED** [review]
added a comment, else the code is clean to understand

**Comment by nishasen** [comment]
Just a design change I want to suggest, you can remove the border of the retry button and keep it transparent, and make the background of the button same as snackbar, else its somewhat looking like three different things.

**Pull request closed without merging**